client/src/document/ingredients/browser-compatibility-table/legend.tsx

```diff
@@ -53,6 +53,9 @@ function getActiveLegendItems(
     for (const [browser, browserSupport] of Object.entries(
       feature.compat.support
     )) {
+      if (browser === "ie") {
```

Comment:
Now that we're having two locations, could we create a constant `HIDDEN_BROWSERS = ["ie"];" in the `index.tsx` and use it in both locations `HIDDEN_BROWSERS.includes(browser)`?

Reply:
> in the `index.tsx`

Do you mean in the `utils.tsx`? I can put this in the `index.tsx`, but the `utils.tsx` is usually where the exports go.

Reply:
Put in `index.tsx` for now.